### ISSUE TITLE
Quick "last month" button for browsing applications

### ISSUE THREAD

**Issue by haldean** [awesomefoundation/awesomebits#251]
Hey folks! I'm a trustee in the San Francisco chapter, and I had a feature request for the application browsing page. When we evaluate grant applications, we always just look at the applications from the last calendar month. It would be super convenient to have a button in this toolbar:

![afsf](https://cloud.githubusercontent.com/assets/187501/22433626/e652b7a4-e6cd-11e6-8a2f-743306c1dc07.png)

...that was a quick "only show last month's applications" button. From lurking in the global awesome mailing list, I realize that every chapter has their own workflow, so I'm mostly filing this to see if there's interest from people outside the SF chapter; if yes, I'm happy to take a stab at an implementation.

**Comment by jcn** [comment]
I'd say that the most generic way of doing this that wouldn't cause problems for people who don't do  things a month at a time would be a link somewhere on the page (or a keyboard shortcut!) that would just pre-fill the start and end dates and optionally submit the form. That way we're not cluttering the page up with a different "search" option.

**Comment by haldean** [comment]
Yeah, something bookmarkable would be nice; maybe a link near the bottom is the way to go?

**Comment by jcn** [comment]
I gotcha - you want a consistent URL where it'll always just show "last month's" applications. That's a neat good idea. I'm not sure the best place to put that, but I would play around with a couple of things - you can just mock them up and we can discuss them in here before submitting a pull request.
